ceph/datadog_checks/ceph/data/conf.yaml.example

```diff
@@ -2,56 +2,57 @@ init_config:
 
 instances:
 
-  ## @param ceph_cmd - string - optional
-  ##
-  #
-  #  ceph_cmd: /usr/bin/ceph
+  -
```

Comment:
is that expected?

Reply:
yup, it's to define at least one instance even if all parameters are optional